src/libcore/mem.rs

```diff
@@ -1131,6 +1151,35 @@ impl<T> MaybeUninit<T> {
     ///
     /// Note that dropping a `MaybeUninit` will never call `T`'s drop code.
     /// It is your responsibility to make sure `T` gets dropped if it got initialized.
+    ///
+    /// # Example
+    ///
+    /// Correct usage of this method: initializing a struct with zero, where all
```

Comment:
```suggestion
    /// Correct usage of this function: initializing a struct with zero, where all
```

Reply:
It *is* a method, though... *shrug*

Reply:
In Rust "method" refers only to functions which can be called as `x.f(y)`. :)

Reply:
Oh right, I didn't realize I copied this wording to `zeroed` as well. Good catch!